### ISSUE TITLE
Create Goal Conflict

### ISSUE THREAD

**Issue by angelleye** [angelleye/GiveWhen#25]
In Divi, I added a new Text module to my page and used the shortcode to drop a goal form onto the page.  When I save the page the shortcode is actually outputting in the admin panel above the Divi modules, partially hidden behind the WP menu, which it's also pushing down below the form.

![image](https://user-images.githubusercontent.com/629241/27850336-923f7f82-6119-11e7-9949-df3933dde86c.png)

**Comment by angelleye** [comment]
Actually, this is happening even when I do it outside of the Divi builder.

**Comment by tejasm-itpath** [comment]
@angelleye  I can't reproduce error you showed in the Image but yes while your adding shortcode inside Divi Builder then it is not displaying properly, And when using Default Editor it displays perfect output

**Comment by angelleye** [comment]
@tejasm-itpath I originally tried this (and pulled the screenshot) via the Divi builder, and I thought it was just a Divi builder conflict.

Then I tried using the default editor instead of Divi builder, and I got the same result in a new page.

So I don't know if it's a Divi theme conflict in general, regardless of whether or not you use the Divi builder, or what exactly is going on, but something is certainly conflicting and causing our shortcode to actually be displayed in the admin panel.
